sympy/matrices/common.py

```diff
@@ -2018,7 +2018,59 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         """
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
+    
+    def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
 
+        Examples
+        ========
+
+        >>> from sympy import MatrixSymbol
+        >>> A = MatrixSymbol('A', 2, 2).as_explicit()
+        >>> A
+        Matrix([
+        [A[0, 0], A[0, 1]],
+        [A[1, 0], A[1, 1]]])
+        >>> A.rot90(k = 0)
```

Comment:
I think that some examples can be moved to `test_common.py`, And have a docstring about the parameter `k`

Reply:
should I do it in a separate pr ?

Reply:
Better in the same PR because it should increase the coverage.

Reply:
Alright ... awesome 😄 ... btw ... you mean testcommonmatrix.py right?